src/modules/partition/partition.conf

```diff
@@ -120,6 +126,8 @@ defaultFileSystemType:  "ext4"
 #           % of the available drive space if a '%' is appended to the value
 #   - minSize: minimum partition size (optional parameter)
 #   - maxSize: maximum partition size (optional parameter)
+#   - features: filesystem features (optional parameter)
+#       name: boolean or integer or string
```

Comment:
"This is only available with KPMCore 4.1.0 and later" should be a documentation somewhere.

Reply:
Done.